sources/packages/backend/apps/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -510,4 +515,53 @@ export class StudentAppealService extends RecordDataModelService<StudentAppeal>
     orderByCondition[dbColumnName] = sortOrder;
     return orderByCondition;
   }
+
+  /**
+   * Get student dependant appeal from student appeal requests and
+   * update student file origin type to Appeal for any uploaded files
+   * @param userId Student user who submits the appeal.
+   * @param studentAppealRequests Payload data.
+   */
+  async updateStudentAppealFiles(
```

Comment:
This whole method could be avoided, if we use the existing utility and send the associated files as paylod from Vue.

![image](https://github.com/bcgov/SIMS/assets/54600590/adfac710-3b02-4e2c-bbf2-5df97cd068d7)